azure/terraform/modules/app_service/app_service.tf

```diff
@@ -1,78 +1,15 @@
-resource "azurerm_app_service" "app_as" {
+resource "azurerm_linux_web_app" "app_as" {
```

Comment:
Why has it changed from azurerm_app_service to azurerm_linux_web_app

Reply:
azurerm_app_service has been deeprecated in the new provider version